src/main/java/sg/ncl/MainController.java

```diff
@@ -822,10 +829,11 @@ public String validateDetails(
             } else {
 
                 teamFields.put("name", signUpMergedForm.getTeamName().trim());
-                teamFields.put("description", signUpMergedForm.getTeamDescription().trim());
-                teamFields.put("website", signUpMergedForm.getTeamWebsite().trim());
-                teamFields.put("organisationType", signUpMergedForm.getTeamOrganizationType());
-                teamFields.put("visibility", signUpMergedForm.getIsPublic());
+                teamFields.put(DESCRIPTION, signUpMergedForm.getTeamDescription().trim());
+                teamFields.put(WEBSITE, signUpMergedForm.getTeamWebsite().trim());
+                teamFields.put(ORGANIZATION_TYPE, signUpMergedForm.getTeamOrganizationType());
+                teamFields.put(VISIBILITY, signUpMergedForm.getIsPublic());
+                teamFields.put("isClass", signUpMergedForm.getIsClass());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "isClass" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)